roles/nginx/vars/main.yml

```diff
@@ -0,0 +1,176 @@
+sites:
+  - servername: www.europython.eu
+    template: europython.conf.j2
+    # This redirects to the main site
+    redirected: https://ep2018.europython.eu
+    server_names:
+      - www.europython.eu
+      - europython.eu
+      - eu.pycon.org
+      - euro.pycon.org
+    root: /var/www/html
+
+  - servername: localhost
+    template: localhost.conf.j2
+    root: /var/www/html
+
+  - servername: secure.europython.eu
+    template: secure.europython.eu.conf.j2
+    server_names:
+      - secure.europython.eu
+      - mail.europython.eu
+      - mattermost.europython.eu
```

Comment:
we are not using mattermost anymore

Reply:
Thanks, I removed the 2 mattermost sites.